RPGLootFeed/Features/ItemLoot/ItemLoot.lua

```diff
@@ -393,8 +393,38 @@ function ItemLoot.Element:new(info, quantity, fromLink)
 						G_RLF.tertiaryToString[TertiaryStats.Indestructible]
 					)
 				end
-				return secondaryText
 			end
+			
+			local alwaysShowSubTypes = { Cloth = true, Leather = true, Mail = true, Plate = true }
+			local equipLocShowSubType = { INVTYPE_WEAPONMAINHAND = true, INVTYPE_WEAPONOFFHAND = true, };
+			local equipLocShowOnlySubType = { INVTYPE_WEAPON = true, INVTYPE_2HWEAPON = true }
+			local extraText = ""
+			if info.itemEquipLoc and info.itemSubType and alwaysShowSubTypes[info.itemSubType] then
+				extraText = " [" .. _G[info.itemEquipLoc] .. " - " .. info.itemSubType .. "]"
+			elseif info.itemEquipLoc and equipLocShowSubType[info.itemEquipLoc] and info.itemSubType then 
+				extraText = " [" .. _G[info.itemEquipLoc] .. " - " .. info.itemSubType .. "]"
+			elseif info.itemEquipLoc and equipLocShowOnlySubType[info.itemEquipLoc] and info.itemSubType then 
+				extraText = " [" .. info.itemSubType .. "]"
+			elseif info.itemEquipLoc then
+				extraText = " [" .. _G[info.itemEquipLoc] .. "]"
+			end
```

Comment:
Maybe this `info.itemEquipLoc` conditional can be part of an outer if-check since it's present and required in every branch. Then you may be able to remove the `extraText` check on Line 411 altogether since it will always be set if `info.itemEquipLoc` is a valid value.